### PR TITLE
Issue: Set up a CSS preprocessor #3

### PR DESCRIPTION
**Pull request by criptodev**
Check the main.sass file, there I put some comments.

### CODE REVIEW

File: css/main.sass [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+$white: #FFFFFF
+$silver: #DDDDDD
+$gray: #AAAAAA
+$black: #111111
+$blue-mid: #0e68b8
+$dark-gray: #1d1d1d
+
+
+
+body
```
Comment (elischutze) [inline css/main.sass]: let's keep it at scss if possible

Comment (criptodev) [inline css/main.sass]: ok!, give me a minute

**Comment by criptodev** [comment]
Its ok now?

File: css/main.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+$white: #FFFFFF;
+$silver: #DDDDDD;
+$gray: #AAAAAA;
+$black: #111111;
+$blue-mid: #0e68b8;
+$dark-gray: #1d1d1d;
+
+
+
+body{
```
Comment (elischutze) [inline css/main.scss]: leave a space between the selector and the brackets for a uniform style

**Review by elischutze — APPROVED** [review]

**Comment by elischutze** [comment]
looks good, merging for #3

**Pull request merged**